musicbot/bot.py

```diff
@@ -1482,7 +1493,12 @@ def check(m):
 
         except ValueError:
             raise exceptions.CommandError('{} is not a valid number'.format(new_volume), expire_in=20)
-
+			
+        if permissions.max_volume and new_volume > permissions.max_volume:
+            raise exceptions.PermissionsError(
+                "[ERROR] You are not able to set the volume above (%s)" % permissions.max_volume
```

Comment:
Probably remove the `[ERROR]` here, it already says `You do not have permission to use this command.` because of the exception that you're raising.